### PR TITLE
Connector Specifications

### PR DESCRIPTION
**Pull request by sparchatus**

### CODE REVIEW

File: connector.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<service android:name="{PushReceiverService}" android:exported="true">
+  <intent-filter>
+    <action android:name="org.unifiedpush.android.action.MESSAGING"/>
+  </intent-filter>
+</service>
+```
+
+### Incoming Handler
+
+The exposed service MUST handle 2 different types of messages:
```
Comment (p1gp1g) [inline connector.md:19]: Did you forget TYPE_DITRIBUTOR_UNREGISTERED, or was it wanted?. The lib can work even if there isn't this type of message so I think it SHOULD handle this one

File: connector.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* TYPE_DISTRIBUTOR_MESSAGE
+* TYPE_DISTRIBUTOR_ENDPOINT_CHANGED
+
+#### TYPE_DISTRIBUTOR_MESSAGE
+
+The distributor sends this type of message to deliver a push message from the provider for the given client.
```
Comment (p1gp1g) [inline connector.md:25]:
```
< The distributor sends this type of message to deliver a push message from the provider for the given client.
> The distributor sends this type of message to forward a push message received from the provider to the end user application.
```

**Review by p1gp1g — CHANGES_REQUESTED** [review]
Thanks :smile: . We need to add the registration part on the connector specification now.

File: connector.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Connector  Library Sepcification
+
+## Unified Messaging Service
+
+The connector library exposes a service to bind to, allowing the distributor to send push related messages.
+
+### Manifest
+The library itself does not declare anything in the manifest. The developer integrating the library should add the following lines to the manifest Wher {PushReceiverService} is the service implementing the below specification.
```
[the remark this answers is not in the document]
Comment (sparchatus) [reply connector.md:8]: Sounds good to me

**Review by p1gp1g — APPROVED** [review]

**Pull request merged**